test/fixtures.py

```diff
@@ -317,21 +336,55 @@ def open(self):
             tries += 1
         else:
             raise Exception('Failed to start KafkaInstance before max_timeout')
-        self.out("Done!")
         self.running = True
+        self.out("Kafka started!")
+
+    def open(self):
+        if self.running:
+            self.out("Instance already running")
```

Comment:
Is there a reason to use `stdout` here rather than `log.info()`? 

I noticed throughout this code that some messages are going to the logs and others going to stdout which seems confusing to me, but perhaps there's a reason or unix convention for this... @dpkp your input?

Reply:
I noticed that as well, but I don't know why. I just kept it the way it had been.
Happy to change it, if needed.

Reply:
@dpkp can you comment here?

Reply:
nudge @dpkp